scripts/components/color-picker-component/color-picker-component.scss

```diff
@@ -0,0 +1,13 @@
+// Inner color picker container.
+.es-color-picker-component__inner-container {
+	padding: 0.5rem 1rem;
+}
+
+// Current color button icon.
+.es-color-picker-component__current-swatch {
+	pointer-events: none;
+	height: 1.15rem;
+	width: 1.15rem;
+	border-radius: 1.5rem;
+	box-shadow: 0 0 0 1px #FFFFFF, 0 0 0 2px #888888;
```

Comment:
Is it too much if I ask that you replace 1px with 0.0625rem and 2px with 0.125rem  🙈 

_please don't hate me_ 😂

Reply:
There are some cases where there are intentional pixel values to avoid floating point rounding errors 😄 
Especially here where you have 2 shadows, 1 on top of the other.